<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QMainWindow" name="TextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_NewFile"/>
    <addaction name="action_OpenFile"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Find"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_NewFile">
   <property name="text">
    <string>New File(N)</string>
   </property>
  </action>
  <action name="action_OpenFile">
   <property name="text">
    <string>Open File(O)</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Save(S)</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>Save As(A)</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit(X)</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>Undo(U)</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>Cut(T)</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>Copy(C)</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>Paste(P)</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="text">
    <string>Find(F)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
